src/agent/onefuzz/src/libfuzzer.rs

```diff
@@ -280,6 +280,10 @@ mod tests {
             .expect("no captures");
 
         assert_eq!(parsed.iters(), 2097152);
-        assert_eq!(parsed.execs_sec(), 699050.0_f64);
+
+        let error_margin = std::f64::EPSILON;
+        let expected: f64 = 699050.0;
+        let execs_sec = parsed.execs_sec();
+        assert!((execs_sec - expected).abs() < error_margin);
```

Comment:
I'd suggest just using the epsilon value directly:
```suggestion
        assert!((execs_sec - expected).abs() < f64::EPSILON);
```
I'll note that clippy suggests using this pattern for approximate equality, but the doc it links to advises AGAINST doing it. 😑